etc/tune_it.sh

```diff
@@ -0,0 +1,112 @@
+#                                                                           #
+# It is not a good idea to e.g: watch high resolution videos while this     #
+# test are running if you do not have enough memory to avoid page faults.   #
+#############################################################################
+
+# Number of rounds overall.
+LIMIT=100
+# Number of loops for each input.
+RLOOPS=10
+# Offset (> 0) . Runs tests with asymmetric input of the form 1:OFFSET
+OFFSET=1
+# Number of positive results (TC-is-faster) accumulated until it is accepted
+# (mainly for "noisy" environments, especially if isolation is not possible)
+LAG=1
+# Keep the temporary file $FILE_NAME. Set to 0 (zero) to remove it at the end.
+KEEP_TEMP=1
+
+echo "You might like to watch the numbers go up to $LIMIT but it will take a long time!"
+
+echo "km ks tc3m tc3s" > $FILE_NAME
+i=1
+while [ $i -le $LIMIT ]; do
+   RNUM=$(LCG)
+   echo $i
+   "$MPWD"/tune -t -r $RLOOPS -L $LAG -S "$RNUM" -o $OFFSET >> $FILE_NAME
+   i=$((i + 1))
+done
+
+i=1
+SUMMARY=""
+TMP=""
+while [ $i -le 4 ]; do
+   if [ $i -eq 1 ]; then
+      v="v"
+   else
+      v=""
+   fi
+   TMP=$(cat  $FILE_NAME | cut -d' ' -f$i | "$MPWD"/statistic_summary_single_column $v;echo z)
+   SUMMARY="$SUMMARY${TMP%z}"
+   i=$((i + 1))
+done
+printf '%s\n' "$SUMMARY"
+if [ $KEEP_TEMP -eq 0 ]; then
+   rm -v $FILE_NAME
+fi
+
+echo "Writing cut-off values to \"bncore.c\" using the median."
+echo "In case of failure: a copy of \"bncore.c\" is in \"bncore.c.orig\""
+
+cp -v $BNCORE_C $BNCORE_C$BACKUP_SUFFIX
+
+cat << END_OF_INPUT > $BNCORE_C
+#include "tommath_private.h"
+#ifdef BNCORE_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis */
+/* SPDX-License-Identifier: Unlicense */
+
+/* Known optimal configurations
+
+ CPU                    /Compiler     /MUL CUTOFF/SQR CUTOFF
+-------------------------------------------------------------
+ Intel P4 Northwood     /GCC v3.4.1   /        88/       128/LTM 0.32 ;-)
+ AMD Athlon64           /GCC v3.4.4   /        80/       120/LTM 0.35
+
+*/
+
+/*
+   Values evaluated on an AMD A8-6600K (64-bit).
+   Please see etc/tune.c for ways to optimize them for your machine but
+   be aware that it may take a long time.
+ */
+END_OF_INPUT
+# No arrays in the Posix shell
+i=1;
+TMP=$(cat $FILE_NAME | cut -d' ' -f$i | "$MPWD"/statistic_summary_single_column m;)
+echo "int KARATSUBA_MUL_CUTOFF = $TMP;" >> $BNCORE_C
+i=$((i + 1))
+TMP=$(cat $FILE_NAME | cut -d' ' -f$i | "$MPWD"/statistic_summary_single_column m;)
+echo "int KARATSUBA_SQR_CUTOFF = $TMP;" >> $BNCORE_C
+i=$((i + 1))
+TMP=$(cat $FILE_NAME | cut -d' ' -f$i | "$MPWD"/statistic_summary_single_column m;)
+echo "int TOOM_MUL_CUTOFF = $TMP;" >> $BNCORE_C
+i=$((i + 1))
+TMP=$(cat $FILE_NAME | cut -d' ' -f$i | "$MPWD"/statistic_summary_single_column m;)
+echo "int TOOM_SQR_CUTOFF = $TMP;" >> $BNCORE_C
+echo "#endif" >> $BNCORE_C
+
+
+
+
+
+
+
+
+
+
```

Comment:
Weird ws

Reply:
So, you think those characters have mental health problems and should go?

Reply:
Indeed

Reply:
If you are wondering what I had to force-push&hellip;